src/test/java/teammates/test/cases/webapi/SaveFeedbackQuestionActionTest.java

```diff
@@ -269,6 +269,34 @@ public void testExecute_invalidQuestionNumber_shouldThrowException() {
                 logic.getFeedbackQuestion(typicalQuestion.getId()).getQuestionDescription());
     }
 
+    @Test
+    public void testExecute_invalidRecommendedLength_shouldThrowException() {
+        InstructorAttributes instructor1ofCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+        FeedbackQuestionAttributes typicalQuestion =
+                logic.getFeedbackQuestion(session.getFeedbackSessionName(), session.getCourseId(), 1);
+
+        loginAsInstructor(instructor1ofCourse1.getGoogleId());
+
+        String[] param = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, typicalQuestion.getFeedbackQuestionId(),
+        };
+
+        FeedbackQuestionSaveRequest saveRequest = getTypicalTextQuestionSaveRequest();
+        FeedbackTextQuestionDetails textQuestionDetails = new FeedbackTextQuestionDetails();
+        textQuestionDetails.setRecommendedLength(-1);
+        saveRequest.setQuestionDetails(textQuestionDetails);
+        SaveFeedbackQuestionAction a = getAction(saveRequest, param);
+
+        assertThrows(InvalidHttpRequestBodyException.class, () -> {
+            getJsonResult(a);
+        });
```

Comment:
You can replace this statement lambda with an expression lambda to make it a little cleaner.

```java
assertThrows(InvalidHttpRequestBodyException.class, () -> getJsonResult(a));
```